zeppelin-web/.jshintrc

```diff
@@ -33,6 +33,6 @@
     "d3": false,
     "BootstrapDialog": false,
     "Handsontable": false,
-    "moment": false
+	"moment" : false
```

Comment:
indention looks a bit off here, although I'm not sure how this change is important to this PR